erpnext/translations/ru.csv

```diff
@@ -921,15 +921,15 @@ Employee,Сотрудник,
 Employee A/C Number,A/C номер сотрудника,
 Employee Advances,Достижения сотрудников,
 Employee Benefits,Вознаграждения работникам,
-Employee Grade,Уровень персонала,
+Employee Grade,Ранг персонала,
 Employee ID,ID сотрудника,
 Employee Lifecycle,Жизненный цикл сотрудников,
 Employee Name,Имя сотрудника,
 Employee Promotion cannot be submitted before Promotion Date ,Повышение сотрудника не может быть выполнено до даты приступления к должности,
 Employee Referral,Перечень сотрудников,
 Employee Transfer cannot be submitted before Transfer Date ,Передача сотрудника не может быть отправлена до даты передачи,
-Employee cannot report to himself.,Сотрудник не может сообщить себе.,
-Employee relieved on {0} must be set as 'Left',"Сотрудник освобожден от {0} должен быть установлен как ""левые""",
+Employee cannot report to himself.,Сотрудник не может сообщить самому себе.,
+Employee relieved on {0} must be set as 'Left',"Сотрудник освобожден от {0} должен быть установлен как ""Покинул""",
```

Comment:
```suggestion
Employee relieved on {0} must be set as 'Left',"Сотрудник освобожден от {0} должен быть установлен как 'Покинул'",
```

Reply:
Thank you, I'll fix such errors in the future

Reply:
@Altizo also if possible please validate CSV syntax and if it's loading correctly on UI from your end. (this and future contributions)

ref: https://github.com/frappe/frappe/pull/16894